### PR TITLE
Update Aliases readme

### PR DESCRIPTION
**Pull request by jwu910**
Hi @jwu910,<br><br>Attached is the fix for [](http://issues.liferay.com/browse/).<br><br>Let me know if you have any questions. Thanks.

### CODE REVIEW

**Comment by jwu910** [comment]
@alexandervalencia

File: aliases/README.md [hunk 1]
```diff
@@ -1 +1,42 @@
-Placeholder for Aliases
\ No newline at end of file
+# Aliases
+The files contained here can be added to your RC files using a variation of the import/include commands.
+We have set up each file to have each alias disabled by default. Unfortunately, this would require you to read through the file to decide what aliases you would like to enable. This is the best way not to interupt your workflow by accidentally overwriting any possible shared alias names.
```
Comment (alexandervalencia) [inline aliases/README.md]: How about, "We have set up each file to have each alias disabled by default. This means you are required to read through the file and decide what aliases you would like to enable. This is the best way not to interupt your workflow by accidentally overwriting any possible shared alias names."

File: aliases/README.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
+
+### Getting started
+#### Setting up your git config file
+1. Read through the `.gitconfig-luitk` file and decide which aliases you would like to enable.
+	- Aliases can be enabled by uncommenting the alias(removing the `#` preceeding the alias)
```
Comment (alexandervalencia) [inline aliases/README.md:8]: "Be sure not to uncomment the actual comments left to help explain what the alias does."

**Review by alexandervalencia — CHANGES_REQUESTED** [review]

**Comment by alexandervalencia** [comment]
🚀  👍

**Pull request merged**